server/routerlicious/packages/services-telemetry/src/lumberjack.ts

```diff
@@ -58,10 +58,45 @@ export class Lumberjack {
         message: string,
         level: LogLevel,
         properties?: Map<string, any> | Record<string, any>,
-        exception?: Error) {
+        exception?: any) {
         this.instance.log(message, level, properties, exception);
     }
 
+    public static debug(
+        message: string,
+        properties?: Map<string, any> | Record<string, any>,
+        exception?: any) {
+        this.instance.log(message, LogLevel.Debug, properties, exception);
+    }
+
+    public static verbose(
+        message: string,
+        properties?: Map<string, any> | Record<string, any>,
+        exception?: any) {
+        this.instance.log(message, LogLevel.Verbose, properties, exception);
+    }
+
+    public static info(
+        message: string,
+        properties?: Map<string, any> | Record<string, any>,
+        exception?: any) {
+        this.instance.log(message, LogLevel.Info, properties, exception);
```

Comment:
Is it actually possible to have exception for Info, Verbose, or Debug? If not, I'd think removing that param would be ideal. Keeping exceptions to warning and error might be a good way to encourage correct LogLevel usage

Reply:
Makes sense!